include/z64camera.h

```diff
@@ -203,26 +203,26 @@ typedef enum {
 
 typedef enum {
     /* 0x00 */ CAM_MODE_NORMAL,
-    /* 0x01 */ CAM_MODE_TARGET, // "PARALLEL"
-    /* 0x02 */ CAM_MODE_FOLLOWTARGET, // "KEEPON"
+    /* 0x01 */ CAM_MODE_Z_PARALLEL, // Holding Z but with no target, keeps the camera aligned
+    /* 0x02 */ CAM_MODE_Z_TARGET_FRIENDLY,
     /* 0x03 */ CAM_MODE_TALK,
-    /* 0x04 */ CAM_MODE_BATTLE,
-    /* 0x05 */ CAM_MODE_CLIMB,
-    /* 0x06 */ CAM_MODE_FIRSTPERSON, // "SUBJECT"
-    /* 0x07 */ CAM_MODE_BOWARROW,
-    /* 0x08 */ CAM_MODE_BOWARROWZ,
-    /* 0x09 */ CAM_MODE_HOOKSHOT, // "FOOKSHOT"
-    /* 0x0A */ CAM_MODE_BOOMERANG,
-    /* 0x0B */ CAM_MODE_SLINGSHOT, // "PACHINCO"
-    /* 0x0C */ CAM_MODE_CLIMBZ,
-    /* 0x0D */ CAM_MODE_JUMP,
-    /* 0x0E */ CAM_MODE_HANG,
-    /* 0x0F */ CAM_MODE_HANGZ,
-    /* 0x10 */ CAM_MODE_FREEFALL,
-    /* 0x11 */ CAM_MODE_CHARGE,
-    /* 0x12 */ CAM_MODE_STILL,
-    /* 0x13 */ CAM_MODE_PUSHPULL,
-    /* 0x14 */ CAM_MODE_FOLLOWBOOMERANG, // "BOOKEEPON"
+    /* 0x04 */ CAM_MODE_Z_TARGET_UNFRIENDLY,
+    /* 0x05 */ CAM_MODE_WALL_CLIMB, // Climbing a wall: ladders and vines
+    /* 0x06 */ CAM_MODE_FIRST_PERSON,
+    /* 0x07 */ CAM_MODE_AIM_ADULT, // First person aiming as adult: bow and hookshot
+    /* 0x08 */ CAM_MODE_Z_AIM, // Third person aiming for all items, child and adult
+    /* 0x09 */ CAM_MODE_HOOKSHOT, // Player being pulled by the hookshot to a target
+    /* 0x0A */ CAM_MODE_AIM_BOOMERANG, // Aiming the boomerang
```

Comment:
maybe `CAM_MODE_HOOKSHOT_PULL` or something to make it clear its not for aiming

Reply:
I did consider that as well. There's still a slight ambiguity as to whether it's an actor being pulled to player vs. player being pulled to an actor (this camera setting is the latter), but still gives more info than just `HOOKSHOT`

Reply:
Another idea is `CAM_MODE_HOOKSHOT_FLY`, as in this is the setting when Link is flying through the air via hookshot?

Reply:
im fine with this yeah